diff -urN boa-0.93.16.1.orig/examples/boa.conf boa-0.93.16.1/examples/boa.conf
--- boa-0.93.16.1.orig/examples/boa.conf	Sat Aug 14 17:10:00 1999
+++ boa-0.93.16.1/examples/boa.conf	Sat Aug 14 17:10:04 1999
@@ -24,8 +24,8 @@
 #  User: The name or UID the server should run as.
 # Group: The group name or GID the server should run as.
 
-User nobody
-Group nogroup
+User http
+Group http
 
 # ServerAdmin: The email address where server problems should be sent.
 # Note: this is not currently used.
@@ -36,28 +36,28 @@
 # with /, it is considered relative to the server root.
 # Set to /dev/null if you don't want errors logged.
 
-ErrorLog /var/log/boa/error_log
+ErrorLog /var/log/httpd/error_log
 
 # AccessLog: The location of the access log file. If this does not
 # start with /, it is considered relative to the server root.
 # Comment out or set to /dev/null (less effective) to disable 
 # Access logging.
 
-AccessLog /var/log/boa/access_log
+AccessLog /var/log/httpd/access_log
 
 # RefererLog: The location of the referer log file. If this does not
 # start with /, it is considered relative to the server root.
 # Comment out or set to /dev/null (less effective) to disable
 # referer logging.
 
-RefererLog /var/log/boa/referer_log
+RefererLog /var/log/httpd/referer_log
 
 # AgentLog: The location of the agent log file. If this does not
 # start with /, it is considered relative to the server root.
 # Comment out or set to /dev/null (less effective) to disable
 # User-Agent logging.
 
-AgentLog /var/log/boa/agent_log
+AgentLog /var/log/httpd/agent_log
 
 # VerboseCGILogs: this is just a logical switch.
 # Comment out to disable.
@@ -72,7 +72,7 @@
 
 # DocumentRoot: The root directory of the HTML documents.
 
-DocumentRoot /var/www
+DocumentRoot /home/httpd/htdocs
 
 # ChRoot: Boa root '/' directory. This is useful to improve security of
 # your system. Don't forget that ALL DIRECTORIES used by boa except logs
@@ -93,12 +93,12 @@
 
 DirectoryIndex index.html
 
-DirectoryMaker /usr/local/sbin/boa_indexer
+DirectoryMaker /usr/sbin/boa_indexer
 
 # LocalCodepage: Local codepage. This is send to client in 'Content-Type:'
 # header by default.
 
-#LocalCodepage iso-8859-1
+#LocalCodepage iso-8859-2
 
 # Codepage: Load codepage conversion table from file. This table will be used
 # on-the-fly conversion.
@@ -161,7 +161,7 @@
 # ScriptAlias: Maps a virtual path to a directory for serving scripts
 # Example: ScriptAlias /htbin/ /www/htbin/
 
-ScriptAlias /cgi-bin/ /usr/lib/cgi-bin/
+ScriptAlias /cgi-bin/ /home/httpd/cgi-bin/
 
 # VirtualHost: Maps a virtual host to a directory.
 # Example: VirtualHost cz.boa.org /html/htdocs/boa/
diff -urN boa-0.93.16.1.orig/src/Makefile.in boa-0.93.16.1/src/Makefile.in
--- boa-0.93.16.1.orig/src/Makefile.in	Sat Aug 14 17:10:00 1999
+++ boa-0.93.16.1/src/Makefile.in	Sat Aug 14 17:11:25 1999
@@ -9,16 +9,13 @@
 
 # Add -DNO_DAEMON if you do not want the server to fork upon launch
 LDFLAGS = @LIBS@
-CFLAGS = -O
+CFLAGS = $(RPM_OPT_FLAGS)
 
 MANDIR = /usr/local/man
 BINDIR = /usr/local/sbin
 
 # FreeBSD's make doesn't like `ifdef`, but likes `.ifdef`
-ifdef INET6
-LDFLAGS += -L/usr/inet6/lib -linet6
-CFLAGS += -DINET6 -I/usr/inet6/include
-endif
+CFLAGS += -DINET6
 
 # Change these if necessary
 
diff -urN boa-0.93.16.1.orig/src/defines.h boa-0.93.16.1/src/defines.h
--- boa-0.93.16.1.orig/src/defines.h	Sat Aug 14 17:10:00 1999
+++ boa-0.93.16.1/src/defines.h	Sat Aug 14 17:10:04 1999
@@ -26,7 +26,7 @@
 /***** Change this, or use -c on the command line to specify it *****/
 
 #ifndef SERVER_ROOT
-#define SERVER_ROOT "/etc/boa"
+#define SERVER_ROOT "/home/httpd"
 #endif
 
 /***** Logs *****************************************************************/
@@ -41,7 +41,7 @@
 #define USE_NLS							 		1
 #define USE_NLS_REFERER_REDIR		1
 #define USE_CHARSET_HEADER   		1
-#define DEFAULT_CHARSET      "iso-8859-1"
+#define DEFAULT_CHARSET      "iso-8859-2"
 
 /****** Authorization ******************************************************/
 
diff -urN boa-0.93.16.1.orig/src/request.c boa-0.93.16.1/src/request.c
--- boa-0.93.16.1.orig/src/request.c	Sat Aug 14 17:10:00 1999
+++ boa-0.93.16.1/src/request.c	Sat Aug 14 17:14:43 1999
@@ -27,10 +27,6 @@
 #include <netinet/tcp.h>
 #include <netdb.h>
 
-#ifdef INET6
-#include <support.h>
-#endif
-
 int sockbufsize = SOCKETBUF_SIZE;
 
 extern int server_s;			/* boa socket */
@@ -154,7 +150,7 @@
 	/* for log file and possible use by CGI programs */
 #ifdef INET6
 	if (getnameinfo((struct sockaddr *)&remote_addr, 
-									NRL_SA_LEN((struct sockaddr *)&remote_addr), 
+									SA_LEN((struct sockaddr *)&remote_addr), 
 									conn->remote_ip_addr, 20,
 								 	NULL, 0, NI_NUMERICHOST)) {
 			fprintf(stderr, "[IPv6] getnameinfo failed\n");
@@ -181,7 +177,7 @@
     if (getsockname(conn->fd, (struct sockaddr *) &salocal, &dummy) == -1)
 									      die(SERVER_ERROR);
 		if (getnameinfo((struct sockaddr *)&salocal,
-                  NRL_SA_LEN((struct sockaddr *)&salocal),
+                  SA_LEN((struct sockaddr *)&salocal),
                   host, 20,
                   NULL, 0, NI_NUMERICHOST)) {
       fprintf(stderr, "[IPv6] getnameinfo failed\n");
diff -urN boa-0.93.16.1.orig/util/Makefile boa-0.93.16.1/util/Makefile
--- boa-0.93.16.1.orig/util/Makefile	Sat Aug 14 17:10:00 1999
+++ boa-0.93.16.1/util/Makefile	Sat Aug 14 17:10:04 1999
@@ -4,10 +4,10 @@
 	rm cpsel cp-test 
 
 cpsel:	cpsel.c	cpsel.config.h
-	gcc -o cpsel cpsel.c
+	gcc $(RPM_OPT_FLAGS) -o cpsel cpsel.c
 
 cp-test: cp-test.c
-	gcc -o cp-test cp-test.c
+	gcc $(RPM_OPT_FLAGS) -o cp-test cp-test.c
 	
 boa_indexer: index_dir.c
-	gcc -o boa_indexer index_dir.c
+	gcc $(RPM_OPT_FLAGS) -o boa_indexer index_dir.c
diff -urN boa-0.93.16.1.orig/util/resolver.pl boa-0.93.16.1/util/resolver.pl
--- boa-0.93.16.1.orig/util/resolver.pl	Sat Aug 14 17:10:00 1999
+++ boa-0.93.16.1/util/resolver.pl	Sat Aug 14 17:10:04 1999
@@ -1,10 +1,10 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # IP address resolver for Boa
 
 # If you want an "in place" change to the log file,
 # change the first line to
-#!/usr/local/bin/perl -i.bak
+#!/usr/bin/perl -i.bak
 # Otherwise, send the output of this program wherever you want:
 #  resolver.pl access_log >access_log_resolved
 
