--- ./src/compat.h.orig	Mon Jul 30 13:44:30 2001
+++ ./src/compat.h	Mon Jul 30 14:16:47 2001
@@ -71,7 +71,7 @@
 
 #ifdef INET6
 #define SOCKADDR sockaddr_storage
-#define S_FAMILY __s_family
+#define S_FAMILY __ss_family
 #define SERVER_AF AF_INET6
 #else
 #define SOCKADDR sockaddr_in
--- ./src/defines.h.orig	Mon Jul 30 13:45:47 2001
+++ ./src/defines.h	Mon Jul 30 13:46:46 2001
@@ -27,7 +27,7 @@
 /***** Change this, or use -c on the command line to specify it *****/
 
 #ifndef SERVER_ROOT
-#define SERVER_ROOT "/etc/boa"
+#define SERVER_ROOT "/etc/httpd"
 #endif
 
 /***** Various stuff that you may want to tweak, but probably shouldn't *****/
--- ./src/ip.c.org      Sat Dec 22 23:29:05 2001
+++ ./src/ip.c  Sat Dec 22 23:33:04 2001
@@ -43,6 +43,7 @@
     */

 #include "boa.h"
+#include "compat.h"
 #include <arpa/inet.h>          /* inet_ntoa */

 /* Binds to the existing server_s, based on the configuration string
@@ -89,8 +90,6 @@
                     dest, len, NULL, 0, NI_NUMERICHOST)) {
         fprintf(stderr, "[IPv6] getnameinfo failed\n");
         *dest = '\0';
-    } else {
-        conn->local_ip_addr = strdup(host);
     }
 #ifdef WHEN_DOES_THIS_APPLY
     if ((s->__ss_family == AF_INET6) &&
--- ./src/request.c.orig	Mon Jul 30 13:48:11 2001
+++ ./src/request.c	Mon Jul 30 13:49:29 2001
@@ -191,7 +191,10 @@
     ascii_sockaddr(&remote_addr, conn->remote_ip_addr, NI_MAXHOST);
 
     /* for possible use by CGI programs */
+    /* the structure doesn't contain port
     conn->remote_port = ntohs(remote_addr.sin_port);
+     */
+    conn->remote_port = 0;
 
     status.requests++;
 
--- ./src/Makefile.in.orig	Mon Jul 30 13:52:33 2001
+++ ./src/Makefile.in	Mon Jul 30 14:11:26 2001
@@ -14,7 +14,8 @@
 
 LDFLAGS = @LDFLAGS@
 LIBS = @LIBS@
-CFLAGS = @CFLAGS@ -I.
+#CFLAGS = @CFLAGS@ -I.
+CFLAGS = @CFLAGS@
 
 # Change these if necessary
 
--- ./boa.conf.orig	Mon Jul 30 13:49:43 2001
+++ ./boa.conf	Mon Jul 30 13:52:00 2001
@@ -45,8 +45,8 @@
 #  User: The name or UID the server should run as.
 # Group: The group name or GID the server should run as.
 
-User nobody
-Group nogroup
+User http
+Group http
 
 # ServerAdmin: The email address where server problems should be sent.
 # Note: this is not currently used, except as an environment variable
@@ -59,7 +59,7 @@
 # Set to /dev/null if you don't want errors logged.
 # If unset, defaults to /dev/stderr
 
-ErrorLog /var/log/boa/error_log
+ErrorLog /var/log/httpd/error_log
 # Please NOTE: Sending the logs to a pipe ('|'), as shown below,
 #  is somewhat experimental and might fail under heavy load.
 # "Usual libc implementations of printf will stall the whole
@@ -71,7 +71,7 @@
 # Comment out or set to /dev/null (less effective) to disable 
 # Access logging.
 
-AccessLog /var/log/boa/access_log
+AccessLog /var/log/httpd/access_log
 # Please NOTE: Sending the logs to a pipe ('|'), as shown below,
 #  is somewhat experimental and might fail under heavy load.
 # "Usual libc implementations of printf will stall the whole
@@ -108,7 +108,7 @@
 # DocumentRoot: The root directory of the HTML documents.
 # Comment out to disable server non user files.
 
-DocumentRoot /var/www
+DocumentRoot /home/httpd/html
 
 # UserDir: The name of the directory which is appended onto a user's home
 # directory if a ~user request is recieved.
@@ -127,7 +127,7 @@
 # DirectoryIndex are commented out, accessing a directory will give
 # an error (though accessing files in the directory are still ok).
 
-DirectoryMaker /usr/lib/boa/boa_indexer
+DirectoryMaker /usr/sbin/boa/boa_indexer
 
 # DirectoryCache: If DirectoryIndex doesn't exist, and DirectoryMaker
 # has been commented out, the the on-the-fly indexing of Boa can be used
@@ -182,5 +182,5 @@
 # ScriptAlias: Maps a virtual path to a directory for serving scripts
 # Example: ScriptAlias /htbin/ /www/htbin/
 
-ScriptAlias /cgi-bin/ /usr/lib/cgi-bin/
+ScriptAlias /cgi-bin/ /home/httpd/cgi-bin/
 
