--- examples/boa.conf	Sun Feb  2 05:02:22 2003
+++ boa.conf	Mon Jul 14 17:50:16 2003
@@ -45,8 +45,8 @@
 #  User: The name or UID the server should run as.
 # Group: The group name or GID the server should run as.
 
-User nobody
-Group nogroup
+User http
+Group http
 
 # ServerAdmin: The email address where server problems should be sent.
 # Note: this is not currently used, except as an environment variable
@@ -155,7 +155,7 @@
 # DocumentRoot: The root directory of the HTML documents.
 # Comment out to disable server non user files.
 
-DocumentRoot /var/www
+DocumentRoot /home/services/httpd/html
 
 # UserDir: The name of the directory which is appended onto a user's home
 # directory if a ~user request is received.
@@ -174,7 +174,7 @@
 # DirectoryIndex are commented out, accessing a directory will give
 # an error (though accessing files in the directory are still ok).
 
-DirectoryMaker /usr/lib/boa/boa_indexer
+DirectoryMaker /usr/sbin/boa_indexer
 
 # DirectoryCache: If DirectoryIndex doesn't exist, and DirectoryMaker
 # has been commented out, the the on-the-fly indexing of Boa can be used
@@ -232,10 +232,10 @@
 # Aliases: Aliases one path to another.
 # Example: Alias /path1/bar /path2/foo
 
-Alias /doc /usr/doc
+Alias /doc /usr/share/doc
 
 # ScriptAlias: Maps a virtual path to a directory for serving scripts
 # Example: ScriptAlias /htbin/ /www/htbin/
 
-ScriptAlias /cgi-bin/ /usr/lib/cgi-bin/
+ScriptAlias /cgi-bin/ /home/services/httpd/cgi-bin/
 
